### ISSUE TITLE
Can join channels with tags that contain spaces, but cannot leave or target them

### ISSUE THREAD

**Issue by turt2live** [Chatterbox/Chatterbox#44]
Added to `channels`:
```
  - name: rf test
    # This is the tag for the channel. It may be used in commands and @-tags to reference this channel. This MAY NOT
    # contains spaces.
    tag: rf test
    # Make this channel permanent so that everyone must be in it.
    permanent: true
```

Can `/ch rf test`, but cannot `/join rf test`. Cannot `/leave rf test` or `@rf test This is a message` (tried variations such as no space and escaping the space). When attempting to leave the channel the message says "You are not in that channel" when you are not in `rf`, but says "Left channel Rythm format" otherwise. Joining has similar behaviour ("Joined channel Rythm format" and "You are already in that channel). `@` messaging sends to the wrong channel or says "You are not in that channel".

**Comment by turt2live** [comment]
Yes, I realize that the documentation says that it cannot contain spaces, but it should not allow `/ch` to work.

**Comment by turt2live** [comment]
Issue not resolved: No complaints in log, can still join channel via `/ch rf test`.

**Comment by turt2live** [comment]
Fixed due to 45e202bb0c553d658c132fcfb0d73f170b27a441
